modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/ImagePins.js

```diff
@@ -100,48 +134,218 @@ const ImagePins = ({
 			zoomIn(containerRef.current, panZoomRef.current);
 		}
 
-		handlers.current = {
-			moveController: (where) =>
-				moveController(
-					containerRef.current,
-					navigationController,
-					where
-				),
-			zoomIn: () => zoomIn(containerRef.current, panZoomRef.current),
-			zoomOut: () => zoomOut(containerRef.current, panZoomRef.current),
+		function dragstarted() {
+			select(this).raise().classed('active', true);
+		}
+
+		function dragged() {
+			select(this).attr(
+				'transform',
+				'translate(' + event.x + ',' + event.y + ')'
```

Comment:
You can use instead of `'translate(' + event.x + ',' + event.y + ')'` -> `translate(`${event.x},${event.y}`)` :)